system/db/barracks.txt

```diff
@@ -3,7 +3,7 @@
 //---------------------------------------------------------------------------
 
 [
-{ mapId: 11, name: "Cozy Lodge", price: 0, characters: 5 },
-{ mapId: 12, name: "Pyromancer Room", price: 5, characters: 7 },
-{ mapId: 13, name: "Banquet Hall", price: 5, characters: 10 },
+{ mapId: 11, name: "Cozy Lodge", price: 0, characters: 4 },
+{ mapId: 12, name: "Pyromancer Room", price: 69, characters: 4 },
+{ mapId: 13, name: "Banquet Hall", price: 69, characters: 4 },
```

Comment:
I seem to recall that each barrack had a larger available number of character slots, but the data file is not showing that.